controls/radwebcam/features/media-information.md

```diff
@@ -55,3 +49,4 @@ To get the available video file formats for the connnected device you can use th
 ## See Also  
```

Comment:
In the Get Video Formats section:
To get the available video file formats for the **connected** device**,** you can use the __RadWebCam.GetVideoFormats__ static method. (With a comma and with only 2 "n"s in connected)